### PR TITLE
Simplify boolean expressions

### PR DESCRIPTION
**Pull request by maacpiash**
Prior to this change, the conditional Boolean values were being set using the ternary operator.

This change removes ternary operators and uses logical operators to simplify the evaluation.

### CODE REVIEW

File: src/darkmode.js [hunk 1]
```diff
@@ -1,17 +1,15 @@
 export default class Darkmode {
   constructor(options) {
```
Comment (jgierer12) [inline src/darkmode.js]:
```suggestion
   constructor(options = {}) {
```

Then you can leave out all the `options &&` checks

File: src/darkmode.js [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
+    const bottom = options && options.bottom || '32px';
+    const right = options && options.right || '32px';
+    const left = options && options.left || 'unset';
+    const time = options && options.time || '0.3s';
+    const mixColor = options && options.mixColor || '#fff';
+    const backgroundColor = options && options.backgroundColor || '#fff';
+    const buttonColorDark = options && options.buttonColorDark || '#100f2c';
+    const buttonColorLight = options && options.buttonColorLight || '#fff';
+    const label = options && options.label || '';
+    const saveInCookies = options && options.saveInCookies;
```
Comment (jgierer12) [inline src/darkmode.js]:
```suggestion
    const saveInCookies = options && !!options.saveInCookies;
```
to ensure it's a boolean

Comment (maacpiash) [reply src/darkmode.js]: In that case, I'd prefer `options && Boolean(options.saveInCookies)`

**Comment by maacpiash** [comment]
@sandoche, what are we going to do about this PR? 🤷‍♂

**Comment by sandoche** [comment]
Hi, just need to test it before merging ! I'll do it this week!

**Pull request merged**